content/index.es.md

```diff
@@ -19,6 +20,12 @@ github:
   branch: gh-pages
   path: content/index.es.md   # Add the language shortcode to the middle of the filename, for example index.fr.md
 
+permalink: /fundamentals/accessibility-intro/es   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
```

Comment:
Permalink, ref, changelog and layout attributes have been duplicated (see line 14 to 17 & line 23 to 27).

Please only keep lines 23 to 27.